worker/index.js

```diff
@@ -193,7 +193,10 @@ async function handleRequest(request) {
 		.map(decodeURIComponent) // for some super special cases, browser will force encode it...   eg: +αあるふぁきゅん。 - +♂.mp3
 		.join('/')
 
-	if (self.props.lite && request.pathname.endsWith('/')) {
+	if (
+		(self.props.lite || request.headers.get('lite') == 'true') &&
```

Comment:
Please change the header name `lite` to `X-Lite`, since it is not a standard HTTP header.

Reply:
Okay, thanks

Reply:
Changed the header name